### PR TITLE
Feature/refactoring

### PR DESCRIPTION
**Pull request by ferokuk**

### CODE REVIEW

File: src/views/BankView.vue [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<template>
+    <div>
+
+    </div>
+</template>
+
+script>
```
Comment (WhisperQFun) [inline src/views/BankView.vue:7]: Need <

File: src/views/ProfileView.vue [hunk 2]
```diff
@@ -1,15 +1,69 @@
...
+    },
+    account:{
+      type: Object,
+      required: true
+    }
+  },
+  async mounted () {
+    this.contract = await ContractPromise
+    this.web3 = w3()
+    this.roles.set('0','Admin')
```
Comment (WhisperQFun) [inline src/views/ProfileView.vue:40]: Move this enum to separate file

**Review by WhisperQFun — CHANGES_REQUESTED** [review]

File: src/views/ProfileView.vue (hunk 2 above)
Comment (ferokuk) [reply src/views/ProfileView.vue:40]: I couldn't :(

File: src/views/BankView.vue (hunk 1 above)
Comment (ferokuk) [reply src/views/BankView.vue]: done

**Pull request merged**